tests/operators/test_snowflake_decorator.py

```diff
@@ -166,10 +176,20 @@ def sample_snow(input_table: Table):
                     warehouse=os.getenv("SNOWFLAKE_WAREHOUSE"),
                 ),
             )
+            x = sample_snow(
+                input_table=f,
+                output_table=Table(
+                    self.snow_inherit_table,
+                    conn_id="snowflake_conn",
+                    schema=os.getenv("SNOWFLAKE_SCHEMA"),
+                    database=os.getenv("SNOWFLAKE_DATABASE"),
+                    warehouse=os.getenv("SNOWFLAKE_WAREHOUSE"),
+                ),
+            )
         test_utils.run_dag(self.dag)
 
         df = hook.get_pandas_df(
-            f'SELECT * FROM "{os.getenv("SNOWFLAKE_DATABASE")}"."{os.getenv("SNOWFLAKE_SCHEMA")}"."{self.snowflake_table}"'
+            f'SELECT * FROM "{os.getenv("SNOWFLAKE_DATABASE")}"."{os.getenv("SNOWFLAKE_SCHEMA")}"."{self.snow_inherit_table}"'
```

Comment:
Do we still have other tests covering the non-inheritance, more basic, use case?